### PR TITLE
bump k8s.io/kubernetes@v1.18.3

### PR DESCRIPTION
**Pull request by andrewsykim**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Pull in the latest v1.18.3 tagged release to get any bug fixes/improvements.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Bump k8s.io/kubernetes to v1.18.3
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,32 +1,31 @@
 module k8s.io/cloud-provider-aws
 
-go 1.12
+go 1.13
 
 replace (
 	// these replacements are pinned to 9e991415386e which is the sha associated with the 1.18.0 tag on k/k
```
Comment (nckturner) [inline go.mod:6]: Probably can update or remove this comment.

Comment (andrewsykim) [reply go.mod]: Good catch, done

**Pull request merged**